dev/io.openliberty.endpoint/src/com/ibm/websphere/channelfw/CFEndPointCriteria.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2003, 2004 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.websphere.channelfw;
+
+/**
+ * The methods of this interface describe requirements needed to
+ * determine an appropriate endpoint to connect to. They are used
+ * in conjunction with request data to determine the appropriate endpoint.
+ * Comparison preferences are chain name first, then virtual host, then
+ * factory classes, then channel accessors, then ssl, and finally local.
+ * Each one narrows down the list of endpoints.
+ */
+public interface CFEndPointCriteria {
```

Comment:
I apologize for the relative lack of clarity in #17980, but I don't think we need to have this class in the new bundle - AFAICT the EndPointMgr/Info implementations don't use this interface